lib/bike_brigade/delivery.ex

```diff
@@ -458,7 +458,9 @@ defmodule BikeBrigade.Delivery do
       |> Repo.preload(:location)
 
     # Does a nested preload to get tasks' assigned riders without doing an extra db query
-    tasks = Repo.preload(all_tasks, assigned_rider: fn _ -> all_riders end)
+    tasks =
+      Repo.preload(all_tasks, assigned_rider: fn _ -> all_riders end)
+    |> Enum.sort_by(fn t -> t.dropoff_location.neighborhood.name end)
```

Comment:
tried to figure out how to do this at the DB level for too long and gave up. Happy to 🍐 if you know a better solution.